app/audit/AuditingService.scala

```diff
@@ -28,6 +28,8 @@ import uk.gov.hmrc.play.audit.AuditExtensions
 import uk.gov.hmrc.play.audit.http.connector.AuditResult.{Disabled, Failure, Success}
 import uk.gov.hmrc.play.audit.http.connector.{AuditConnector, AuditResult}
 import uk.gov.hmrc.play.audit.model.{DataEvent, ExtendedDataEvent}
+import play.api.libs.json.JodaWrites._
```

Comment:
These particular imports are weird in that they don't appear to be used but if they are removed then it can't compile. I included them as suggested by the upgrade guide.

Would it be worth me adding a comment to explain this?